src/hooks/useEditState.js

```diff
@@ -22,46 +22,62 @@ export const useEditState = (entityId, sectionName, extendedValidationRules = {}
 
 	const mergedValidationRules = { ...validationRules, ...extendedValidationRules };
 
-	const useFieldState = (keys, initialValue = "", errorTypes = [], saveInitialValueToEditState = false) => {
+	const useFieldState = (
+		keys,
+		initialValue = "",
+		errorTypes = [],
+		saveInitialValueToEditState = false,
+		validateInitialValueAfterSave = false,
```

Comment:
Why you need those changes instead of just using dynamic edit state which allows you to do that? Technically dependencies means that there are dynamic relations so, it make sense to use dynamic edit state for that. Previously we already discussed it with team and it was decided to use dynamic edit state instead of modify static.
Regarding validateInitialValueAfterSave - I dont understand why you need that field? You can call isValid from the app to validate edit state when its necessary. Having this field makes that behavior redundant